ops/cloud-deployment/main.bicep

```diff
@@ -1,241 +1,214 @@
+@description('Stack Name that helps give many resources unique names. Flexion and USTP Required')
 param appName string
 param location string = resourceGroup().location
-
-@description('Disable creating Azure virtual network by default.')
+param appResourceGroup string = resourceGroup().name
+@description('Boolean to deploy vnet. Determined at workflow runtime. Flexion and USTP required on initial deployment')
 param deployVnet bool = false
-@description('Create Azure virtual network. Set flag to false when vnet already exists.')
-param createVnet bool = false
+@description('Prefic to Vnet address. USTP Required')
 param vnetAddressPrefix array = [ '10.10.0.0/16' ]
 
-@description('Setup Azure resources for Private DNS Zone. Link virtual networks to zone.')
+@description('Setup Network Resources: DNS Zone, Subnets, Link virtual networks to zone. Flexion and USTP required')
 param deployNetwork bool = true
+@description('Network RG Name. Flexion and USTP Required')
 param networkResourceGroupName string
+@description(' Vnet Name. USTP Required')
 param virtualNetworkName string = 'vnet-${appName}'
+@description('Array of Vnets to link to DNS Zone. USTP Required')
 param linkVnetIds array = []
 
-@description('Set to true to deploy web module resources. This should be set to false for Azure slot deployments.')
+@description('Private Endpoint Subnet Name. USTP Required')
+param privateEndpointSubnetName string = 'snet-${appName}-private-endpoints'
+@description('Private Endpoint Subnet address prefix. USTP Required')
+param privateEndpointSubnetAddressPrefix string = '10.10.12.0/28'
```

Comment:
I don't really see value in descriptions like this. If we were to put this CIDR in a variable or move it to the GitHub Workflow, we wouldn't need the `USTP Required` part because it would always be required, and the rest of the description is redundant.

Reply:
That is a fair statement. I had added these descriptions prior to reevaluating the azure-deploy.sh script, which also detracts from the value of these.